File: src/main/java/jp/artan/japanesefoodmod/init/tabs/ToolCreativeTab.java
package jp.artan.japanesefoodmod.init.tabs;

import jp.artan.japanesefoodmod.raw_materials.JapaneseFoodRawMaterials;
import net.minecraft.creativetab.CreativeTabs;
import net.minecraft.item.ItemStack;

public class ToolCreativeTab  extends CreativeTabs {

    public ToolCreativeTab() {
        super("japanesefoodtool");
    }

    @Override
    public ItemStack getTabIconItem() {
        return new ItemStack(JapaneseFoodRawMaterials.GROUND_WOOD);
    }

}

File: src/main/java/jp/artan/japanesefoodmod/init/tabs/ToolCreativeTab.kt
package jp.artan.japanesefoodmod.init.tabs

import jp.artan.japanesefoodmod.init.ItemInit
import net.minecraft.creativetab.CreativeTabs
import net.minecraft.item.ItemStack

class ToolCreativeTab : CreativeTabs("japanesefoodtool") {


    override fun getTabIconItem(): ItemStack {
        return ItemStack(ItemInit.GROUND_WOOD)
    }

}